src/qt/qtipcserver.h

```diff
@@ -1,4 +1,10 @@
+#ifndef QTIPCSERVER_H
+#define QTIPCSERVER_H
+
+// Define Bitcoin-Qt message queue name
 #define BITCOINURI_QUEUE_NAME "BitcoinURI"
```

Comment:
I'd expect this to be PaycoinURI but being that the URI stuff is in a separate branch you probably have already changed that there.